boards/x86/up_squared/up_squared.yaml

```diff
@@ -0,0 +1,11 @@
+identifier: up_squared
+name: UP Squared
+type: mcu
```

Comment:
is mcu is the right name for Apollo lake?

Reply:
scripts/sanity_chk/sanitycheck-platform-schema.yaml does not seem to have any other suitable type. Do you want to add one?